setup.cfg

```diff
@@ -7,8 +7,13 @@ cover-min-percentage=100
 cover-erase=true
 
 [flake8]
+exclude=./docs/conf.py,./test/*,./urllib3/packages/*
 max-line-length=99
 
+[pep8]
```

Comment:
Through your narky comment, I get this very strong sense that you consider that duplication to be more important than allowing pep8 to be used directly.  No worries.  As I said, this is not needed, and I'll remove it in the next update.